src/redis-check-aof.c

```diff
@@ -429,6 +429,19 @@ input_file_type getInputFileType(char *filepath) {
     }
 }
 
+void checkAndPrintResult(int ret, char *aofFileName, char *aofType) {
```

Comment:
```suggestion
void printAofStyle(int ret, char *aofFileName, char *aofType) {
```

Reply:
Thank you. The function name has been modified. There are three calls corresponding here, which are different everywhere, and cannot be well matched when placed in checkSingleAof.